R/survey_statistics.r

```diff
@@ -363,11 +360,10 @@ survey_quantile <- function(
   if (!is.null(vartype)) {
     vartype <- if (missing(vartype)) "se" else match.arg(vartype, several.ok = TRUE)
   }
-  if (missing(interval_type) & !inherits(.svy, "svyrep.design")) interval_type <- "Wald"
-  if (missing(interval_type) & inherits(.svy, "svyrep.design")) interval_type <- "probability"
+  if (missing(interval_type) & !inherits(.svy, "svyrep.design")) interval_type <- "mean"
+  if (missing(interval_type) & inherits(.svy, "svyrep.design")) interval_type <- "mean"
```

Comment:
Now that all types of surveys use "mean" as default, collapse to a single line: 
```
if (missing(interval_type)) interval_type <- "mean"
```

Reply:
Glad you noticed that. https://github.com/gergness/srvyr/pull/124/commits/f1a4cd92f8c8c03e9766495b5e0299ef6fc77d7b ought to fix this.